src/platforms/android/troubleshooting.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve common issues with the Android SDK."
+sidebar_order: 2000
+---
+
+## NDK
+
+If you are using `sentry-android` version earlier than 5.0, due to current limitations in how `sentry-native` works, it cannot identify native libraries loaded directly from `.apk` files. Instead, it needs to have access to the extracted libraries on the file system.
+This functionality is [disabled by default](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs)
+when using an Android Gradle Plugin >= v3.6.0, and it needs to be enabled with the
+[`extractNativeLibs`](https://developer.android.com/guide/topics/manifest/application-element#extractNativeLibs) configuration option.
+
+If you're using [Android App Bundle](https://developer.android.com/guide/app-bundle), set `android.bundle.enableUncompressedNativeLibs=false` along with the `extractNativeLibs` configuration option.
+
+Apart from this, `sentry-native` also uses [tagged pointers](https://source.android.com/devices/tech/debug/tagged-pointers)
+internally. As a result, the Android SDK will not work out of the box on newer Android devices which have their own conflicting pointer tagging.
```

Comment:
Do we mean the SDK won't work on: 
1. specifically newer Android devices that have this conflicting pointer tagging
OR
2. all newer Android devices because they all have this conflicting pointer tagging

Let me know which, and I can add either add a comma or change a word to clarify this.

```suggestion
internally. As a result, the Android SDK will not work out of the box on newer Android devices which have their own conflicting pointer tagging.
```

Reply:
Option 2, and it's not about newer Android devices, but rather Android devices with newer Android Operating system

Reply:
Gotcha. How about this:
```suggestion
internally. As a result, the Android SDK will not work out of the box on devices with a newer Android OS because they have their own conflicting pointer tagging.
```